lib/package-generator-view.coffee

```diff
@@ -89,6 +97,9 @@ class PackageGeneratorView extends View
 
     @runCommand(atom.packages.getApmPath(), args, callback)
 
+  isInPackageMode: ->
```

Comment:
Is this necessary? `if @mode is 'package'` seems fine to me, and it's also similar to what vim-mode does.